target-tests/README.md

```diff
@@ -0,0 +1,58 @@
+# Target Tests
+
+*Hardware in the loop (HIL) test suite for the
+[pa-spl](https://github.com/EngJay/pa-spl) driver.*
+
+## Running the Tests
+
+The HIL tests are configured to use a 0x32 or 0x33 version of the PCB Artists
+I2C SPL Module connected to the I2C1 interface of a STM32F3 Discovery
+development board.
+
+Set up the hardware:
+
+- connect the SCL, SDA, 3.3V, and ground pins of the sensor to pins PB6, PB7,
+  a 3.3V, and a ground pin on the dev board, respectively.
+- connect the dev board to the machine running the tests via its ST-Link USB
+  connection.  
+
+Then, run the tests with cargo, to which something like this response should be
```

Comment:
Then, run the tests with cargo. The received response is expected to be similar to the following:

Reply:
Fixed.